cbmc/proofs/IotMqtt_strerror/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16 
+cbmcflags:        '=--unwind;1;--bounds-check;--pointer-check;--unwinding-assertions;--nondet-static=' 
```

Comment:
Seems like you're missing flags?

Reply:
I forgot to update the `cbmc-batch.yaml` file.